.bazelrc

```diff
@@ -1,6 +1,7 @@
 # Must be first. Enables build:windows, build:linux, build:macos, build:freebsd, build:openbsd
 build --enable_platform_specific_config
-
+build --local_ram_resources=HOST_RAM*.7 --local_cpu_resources=4
+build --disk_cache=~/bazel-cache
```

Comment:
Is this intentional?

Reply:
Oops, I'll remove these. They are only used locally.